frontend/src/Registration.js

```diff
@@ -1,8 +1,7 @@
 import React, { Component } from 'react';
-import { Link, Redirect } from 'react-router-dom';
-
-import { loginRequester, registerRequester } from './requests/requestBuilder';
-import './styles/Registration.css';
+import { Link } from 'react-router-dom';
+import './styles/main.css';
```

Comment:
What is this import for?

Reply:
the link is the sign in button.
since registration.css and login.css are similar, I merged the two into main.css.